### DISCUSSION TITLE
I've made my own dialogic

### DISCUSSION THREAD

**Discussion by portaloffreedom** [dialogic-godot/dialogic#1519] [Show and tell]
Hi!
I've used Dialogic in a previous game jam and I quite enjoyed the structure that it gave.

There were a few pain points, like only GUI editing of the scripts and everything saved in huge non git friendly files.
So I've started creating my own dialog scripting system, with his own quirks. It's in gdextension, so native c++, with a real interpreter, something more similar to [ink](https://www.inklestudios.com/ink/) or [yack](https://www.grumpygamer.com/loom). It's quite a different beast and I hope there is enough space for both projects. 

At the moment it's a very early development process, and it's going to take some time (if I hold my energy and passion for long enough) before it reaches quality levels similar to Dialogic.

I'm posting here because my code is heavily inspired from Dialogic, with lots of copied code from the "rendering" code, including all scenes and the assets. Of course if there is anything you like, you are welcome to copy it back.

If you want to check it out and give some feedback, it's here: https://codeberg.org/portaloffreedom/gdialog

**Comment by coppolaemilio** [comment]
Thanks for sharing! good luck with the plugin and looking forward to seeing it in action :)
